src/gtk-4.0/widgets/_shortcuts.scss

```
@@ -0,0 +1,42 @@
+shortcuts-section {
+    margin: rem(12px);
+}
+
+shortcuts-group > label {
+    @extend .h4;
+}
+
+// Remove plus signs in Gtk.ShortcutLabel
+shortcut .dim-label {
+    color: transparent;
```

Do we also want to change the font size here so the spacing between keycaps iisn't determined by the invisible `+`?

---

We could! One character width seems to be a good spacing already to me, but I'm open to suggestions

---

Okay, maybe this is a hack and not worth it, but then again hiding the `+` in the first place feels like a hack, so… :shrug: We use 6px column spacing in our shortcut overlay, for example:

```suggestion
    color: transparent;
    font-size: 0;
    margin-left: rem(3px);
    margin-right: rem(3px);
```

---

Would it make more sense to just make the font size rem(6px)?

---

@danrabbit probably not because the `+` character isn't guaranteed to actually be `1em` in size, so it wouldn't match anyway. But see my requested changes; I don't think we want to actually fiddle with the styling of dim labels, anyway, due to how they're actually used.